docs/minimal/transformations.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: default
+title: Custom Transformation
+parent: Minimal downtime migrations
+nav_order: 4
+---
+
+# Minimal downtime migrations for MySQL
```

Comment:
Seems like this needs to be broken down into two parts - 

1. How to Implement a custom transform. (Implement, Unit test?)
3. How to use the custom transform in a pipeline (forward or reverse).

Currently does not read well IMO.

Reply:
We should also put the following contents in a separate page and make it agnostic of minimal downtime migration (because bulk docs will refer to these) :
- Building a JAR
- Parameter details
- Java Object type mappings
Anything else that is not minimal downtime specific

Reply:
Broke it down into multiple sections and moved it to a common place